js/src/forum/index.js

```diff
@@ -21,12 +21,14 @@ const styles = {
   'header': { prefix: '### ' },
   'bold': { prefix: '**', suffix: '**', trimFirst: true },
   'italic': { prefix: '_', suffix: '_', trimFirst: true },
+  'strikethrough': { prefix: '~~', suffix: '~~', trimFirst: true },
   'quote': { prefix: '> ', multiline: true, surroundWithNewlines: true },
   'code': { prefix: '`', suffix: '`', blockPrefix: '```', blockSuffix: '```' },
   'link': { prefix: '[', suffix: '](https://)', replaceNext: 'https://', scanFor: 'https?://' },
   'image': { prefix: '![', suffix: '](https://)', replaceNext: 'https://', scanFor: 'https?://' },
   'unordered_list': { prefix: '- ', multiline: true, surroundWithNewlines: true },
-  'ordered_list': { prefix: '1. ', multiline: true, orderedList: true }
+  'ordered_list': { prefix: '1. ', multiline: true, orderedList: true },
+  'spoiler': { prefix: '>!', suffix: '!<', blockPrefix: '>! ', multiline: true, trimFirst: true }
```

Comment:
Ooo I like that block and regular spoilers could be handled by one button. Any chance you could include a video of this in action?

Reply:
Sorry but that's not possible atm because of my quota. -and hey, hope you're doing well, miss you!